lock-api/src/main/java/com/palantir/lock/LockDescriptor.java

```diff
@@ -48,25 +48,30 @@
     }
 
     @JsonIgnore
+    @SuppressWarnings("checkstyle:RegexpSinglelineJava") // JDK StandardCharsets is only a feature in 1.7+
     public String getLockIdAsString() {
         return new String(bytes, Charsets.UTF_8);
     }
 
     @Override
-    public int compareTo(LockDescriptor o) {
-        return UnsignedBytes.lexicographicalComparator().compare(this.bytes, o.bytes);
+    public int compareTo(LockDescriptor other) {
+        return UnsignedBytes.lexicographicalComparator().compare(this.bytes, other.bytes);
     }
 
     @Override
     public String toString() {
         String lockIdAsString = getLockIdAsString();
-        return getClass().getSimpleName() + " [" +
-                (BASIC_PRINTABLE_ASCII.matchesAllOf(lockIdAsString) ?
-                        lockIdAsString :
-                        BaseEncoding.base16().encode(bytes))
+        return getClass().getSimpleName() + " ["
+                + getLockIdString(lockIdAsString)
                 + "]";
     }
 
+    private String getLockIdString(String lockIdAsString) {
```

Comment:
I find it a bit confusing having `getLockIdAsString()` and `getLockIdString(String lockIdAsString)`.  At a first glance I don't see the difference in the method names, and when I do it doesn't communicate the reason for the 2 methods.

Is it still right to have the 2 methods?  If we do can we rename this to something like `encodeToAscii(String s)`?